Scripts/create_cache.sh

```diff
@@ -28,7 +28,7 @@ while getopts "fc" option ; do
         rm -Rf ${cacheDir}
         echo "Cache dir ${cacheDir} cleared...";;
     c ) # use custom wallpaper
-        inWall="$OPTARG"
+        inWall="${2:-${OPTARG}}"
```

Comment:
Hey thanks, but don't use the $2 parameter here, it should just fail is wallpaper is not passed.